c++/src/kj/exception.c++

```diff
@@ -19,19 +19,19 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+#include <stdlib.h>
```

Comment:
I don't see any build errors on Alpine 3.4 with the ordering unchanged, so not sure there is an issue.